src-tauri/src/file_utils.rs

```diff
@@ -2,6 +2,7 @@ extern crate fs_extra;
 
 use std::fs;
 use std::path::Path;
+use tauri::api::dialog::blocking::FileDialogBuilder;
```

Comment:
Burde vi ikke bruke non-blocking varianten? Eller er det en grunn til at vi vil blokke?

Reply:
Slik jeg har satt det opp starter dialogen i en egen thread med en blocking dialog, så ikke main-thread blir brukt. 

Var etter jeg leste [denne kommentaren](https://stackoverflow.com/a/76140801/12321920) fra FabianLars selv at jeg gjorde det, men siden async [er anbefalt](https://tauri.app/v1/guides/features/command/#async-commands) for ikke å skape unødvendige tregheter i UIet, og siden det med async er [anbefalt å bruke blocking](https://docs.rs/tauri/latest/tauri/api/dialog/blocking/index.html) tenkte jeg det var like greit å gjøre det slikt.